Assets/Scripts/Player/PlayerControls.cs

```diff
@@ -229,7 +285,15 @@ public override void Update()
 				}
 				else
 				{
-					player.physics.ChangeState( PhysicsStateType.Grounded );
+					if ( player.controls.jumpButton.down &&
```

Comment:
Do we want it to be possible to jump during this frame b/w gliding and landing? Seems weird but I guess it doesn't hurt